presto-hive/src/main/java/com/facebook/presto/hive/authentication/KerberosHiveMetastoreAuthentication.java

```diff
@@ -60,7 +66,7 @@ public TTransport authenticate(TTransport rawTransport, String hiveMetastoreHost
                     "Kerberos principal name does NOT have the expected hostname part: %s", serverPrincipal);
 
             Map<String, String> saslProps = ImmutableMap.of(
-                    Sasl.QOP, "auth",
+                    Sasl.QOP, hdfsWireEncryptionEnabled ? "auth-conf" : "auth",
```

Comment:
Do you think we should have a separate property for metastore traffic encryption? e.g.: `hive.metastore.wire-encryption.enabled`